schema/project.js

```diff
@@ -192,7 +192,7 @@ class Project extends BaseModel {
       .leftJoinRelation('licenceHolder')
       .withGraphFetched('[licenceHolder, additionalEstablishments(constrainEstParams), establishment(constrainEstParams)]')
       .modifiers({
-        constrainEstParams: builder => builder.select('id', 'name')
+        constrainEstParams: builder => builder.select('id', 'name', 'status')
```

Comment:
Is that not going to give us the status of the PEL?

Reply:
```
     DBError: select "project_establishments"."project_id" as "objectiontmpjoin0", "id", "name", "status" from "establishments" inner join "project_establishments" on "establishments"."id" = "project_establishments"."establishment_id" where "establishments"."deleted" is null and "project_establishments"."project_id" in ($1) - column reference "status" is ambiguous```

Reply:
sounds like yes